docs/source/spatialstats.rst

```diff
@@ -0,0 +1,108 @@
+In particular, these tools help to:
+
+- account for non-stationarities of elevation measurement errors (e.g., varying precision of DEMs with terrain slope),
+- quantify the spatial correlation of measurement errors in DEMs (e.g., native spatial resolution, instrument noise),
+- estimate robust errors for observations integrated in space (e.g., average or sum of samples),
+- propagate errors between spatial ensembles at different scales (e.g., sum of glacier volume changes).
+
+.. contents:: Contents 
+   :local:
+
+Assumptions for statistical inference in spatial statistics
+***********************************************************
+
+Spatial statistics are valid if the variable of interest verifies the assumption of stationarity of the 1\ :sup:`st` and 2\ :sup:`nd` orders.
+That is, if the two following assumptions are verified:
+
+1. The mean of the variable of interest is stationary in space, i.e. constant over sufficiently large areas,
+2. The variance of the variable of interest is stationary in space, i.e. constant over sufficiently large areas.
+
+A sufficiently large averaging area is an area expected to fit within the spatial domain studied.
+
+In other words, for a reliable analysis, the DEM should:
+
+1. Not contain systematic biases that do not average out over sufficiently large distances (e.g., shifts, tilts), but can contain pseudo-periodic biases (e.g., along-track undulations),
+2. Not contain measurement errors that vary significantly in space.
+
+Precision of a single DEM, or of a difference of elevation data
+***************************************************************
+
+To infer the precision of a DEM, it is compared against other elevation data.
+If the other elevation data is known to be of higher-precision, one can assume that the analysis of differences will represent the precision of the rougher DEM.
+Otherwise, the difference will describe the precision with significant measurement errors originating from both the DEM and the other dataset.
+
+TODO: complete with Hugonnet et al. (in prep)
+
+Stable terrain: proxy for infering DEM precision
+************************************************
+
+TODO
+
+Metrics for DEM precision
+*************************
+
+Pixel-wise elevation measurement error
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
+
+
+Spatially-integrated elevation measurement error
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The standard error (SE) of a statistic is the standard deviation of the distribution of this statistic.
+For spatially distributed samples, the standard error of the mean (SEM) is of great interest as it allows quantification of the error of a mean (or sum) of samples in space.
+
+The standard error  :math:`\sigma_{\overline{dh}}` of the mean :math:`\overline{dh}` of elevation changes samples :math:`dh` is typically derived as:
+
+.. math::
+
+        \sigma_{\overline{dh}} = \frac{\sigma_{dh}}{\sqrt{N}},
+
```

Comment:
Could this be suffixed by a python version of the equation? Algebra is just another programming language, right? ;)

Something like:
```python
mean_stderr = dispersion / np.sqrt(n_observations)
```

I think it would be nice to consistently have both algebra and python representations. The conciseness of algebra can often be quite daunting, especially for newcomers. But its universal nature makes it great to have, so a compromise would be to always do both. What do you think?

Reply:
I prefer writing the algebra formulation which is universal. 
But an option could be to point out to the line of code where it is implemented. I don't know how easy it is to track the code line when the code changes though.

Reply:
Amaury's solution seems good, maybe we could have a file in /source/code that has all the equations in Python we can point to? 
Not sure it should be tested by `test_docs.py`, though. We could place it in another folder: source/equations or similar?